content/en/real_user_monitoring/session_replay/mobile/privacy_options.md

```diff
@@ -410,9 +408,148 @@ With the `allow` setting enabled, all text is revealed.
 {{% /tab %}}
 {{< /tabs >}}
 
+## Privacy Overrides
+
+The sections above describe the global masking levels that apply to the entire application. However, it is also possible to override these settings at the view level. The same privacy levels as above are available for text and inputs, images, touches, and an additional setting to completely hide a specific view.
+
+To ensure overrides are recognized properly, they should be applied as early as possible in the view lifecycle. This prevents scenarios where Session Replay might process a view before applying the overrides.
+
+Privacy overrides affect views and their descendants. This means that even if an override is applied to a view where it might have no immediate effect (e.g., applying an image override to a text input), the override still applies to all child views. 
+
+Overrides operate using a "nearest parent" principle: if a view has an override, it uses that setting; otherwise, it inherits the privacy level from the closest parent in the hierarchy with an override. If no parent has an override, the view defaults to the application's general masking level.
```

Comment:
```suggestion
Overrides operate using a "nearest parent" principle: if a view has an override, it uses that setting. Otherwise, it inherits the privacy level from the closest parent in the hierarchy with an override. If no parent has an override, the view defaults to the application's general masking level.
```